### DISCUSSION TITLE
Getting the current value on parameter changes to do incremental parameter changes

### DISCUSSION THREAD

**Discussion by HorstBaerbel** [vcync/modV#856] [Q&A] [answered]
Say you want to change the rotation of an object in an increment every time audio:kick occurs ("Input Config"->"Expression"). Is this currently possible in modV? I see no option to get the _current value_ of the parameter I want to change.

**Comment by 2xAA** [comment, marked as answer]
@HorstBaerbel Currently the value of the parameter isn't exposed to the scope of the expression. That's a good feature however and should be somewhat straightforward to implement.

I'll transfer this to a feature issue, thanks.

**Reply by 2xAA** [reply]
#857 created from this discussion.
